client/src/pages/Home.js

```diff
@@ -5,7 +5,7 @@ const Home = () => {
   return (
     <div className={styles.home}>
       <VideoForm />
-      <IFrame youtubeID="MwDYOJNlljE" start="2" end="7" time="3" />
```

Comment:
- Need to find other way to keep <Iframe> in Home.js

Reply:
Fixed! Check commit below!